nextflow_schema.json

```diff
@@ -1,358 +1,320 @@
 {
-    "$schema": "http://json-schema.org/draft-07/schema",
-    "$id": "https://raw.githubusercontent.com/epi2me-labs/wf-basecalling/master/nextflow_schema.json",
-    "title": "epi2me-labs/wf-basecalling",
-    "description": "Helper workflow for basecalling ONT reads.",
-    "demo_url": "https://ont-exd-int-s3-euwst1-epi2me-labs.s3.amazonaws.com/wf-basecalling/wf-basecalling-demo.tar.gz",
```

Comment:
These lines are required by our platform to enable demo functionality to work. They will need to be reinstated.